index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    
+    <link rel="stylesheet" href="src/css/style.css" />
+    <link rel="stylesheet" href="src/css/top-section.css" />
+    <link rel="stylesheet" href="src/css/mid-section.css" />
+    <link rel="stylesheet" href="src/css/guests-section.css" />
+    <link rel="stylesheet" href="src/css/partners.css" />
+    <script
+      src="https://kit.fontawesome.com/a4bb0e0fbd.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Gitex Africa</title>
+  </head>
+  <body>
+    <section class="header">
+      <header>
+        <nav class="nav">
+          <div class="mobile-menu">
+            <i class="fa-sharp fa-solid fa-bars"></i>
+          </div>
+          <div class="menu-mobile hideMenu">
+            <button class="close"><h2>X</h2></button>
```

Comment:
- [ ] [OPTIONAL] There's something funny going on with the close menu button here please check the video below 👇
  ![firefox_vd9TroorKi](https://user-images.githubusercontent.com/104968236/234911427-c94cebf5-589b-4766-a8b6-6d5a50c7a8d2.gif)
- Please make sure you improve it as it leads to a bad UX